.circleci/config.yml

```diff
@@ -0,0 +1,99 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/go/src/github.com/docker/swarmkit
+    environment:
+      # Needed to install go
+      OS: linux
+      ARCH: amd64
+      GOVERSION: 1.10.2
+      # Needed to install protoc
+      PROTOC: https://github.com/google/protobuf/releases/download/v3.5.0/protoc-3.5.0-linux-x86_64.zip
+
+    # Note(cyli): We create a tmpfs mount to be used for temporary files created by tests
+    # to mitigate the excessive I/O latencies that sometimes cause the tests to fail.
```

Comment:
Is the latency due to files being written in the container's (CoW) filesystem, or just due to physical disk instead of memory (tmpfs) being used? Wondering if a `VOLUME /some/path` would be resolving the latency sufficiently.

Reply:
We did this in CircleCI V1 as well due to those issues, so possibly just whatever is running the CI's disk may be too slow?